togglwrapper/mixins.py

```diff
@@ -35,6 +35,45 @@ def get(self, id=None, child_uri=None, params=None):
         uri = self._compile_uri(id, child_uri=child_uri)
         return self.toggl.get(uri, params=params)
 
+    def _valid_lower_values(self, value, name, allowed):
```

Comment:
Being strict about only accepting either `bool`s or the string "both" might make these helper functions unnecessary, but if they're kept, I'd suggest moving out into their own module, to keep this one focused on Mixin classes.